pkg/openstack/loadbalancer.go

```diff
@@ -352,6 +352,7 @@ type serviceConfig struct {
 	healthMonitorTimeout        int
 	healthMonitorMaxRetries     int
 	healthMonitorMaxRetriesDown int
+	canUseHTTPMonitor			bool
```

Comment:
Why is this necessary? It's odd that a change to the code needs to be made to allow unit testing.